vnext/Microsoft.ReactNative/QuirkSettings.cpp

```diff
@@ -30,9 +24,52 @@ winrt::Microsoft::ReactNative::ReactPropertyId<bool> MatchAndroidAndIOSStretchBe
   properties.Set(MatchAndroidAndIOSStretchBehaviorProperty(), value);
 }
 
-/*static*/ bool QuirkSettings::GetMatchAndroidAndIOSStretchBehavior(
-    winrt::Microsoft::ReactNative::ReactPropertyBag properties) noexcept {
+winrt::Microsoft::ReactNative::ReactPropertyId<bool> UseLegacyWebSocketModuleProperty() noexcept {
+  static winrt::Microsoft::ReactNative::ReactPropertyId<bool> propId{L"ReactNative.QuirkSettings",
```

Comment:
nit: keeping a static propId doesn't avoid the copy here, since we do a copy of it for the return. You can just return the props without caching a static one.

Same below

Reply:
Noted. Assumed the usage was valid based on other settings.